index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const bookbatches = require("./bookbatches.router.js");
+const featured = require("./featured.router.js");
+const userdata = require("./login.router.js");
+const userauth = require("./authcheck.router.js");
+var cors = require('cors');
+const PORT= 3000;
+
+const uri = `${process.env['uri']}`;
+
+
+mongoose.connect(uri , {
+  useNewUrlParser : true,
+  useUnifiedTopology : true
+})
```

Comment:
Hey,
Try exporting this function in a separate filer altogether, and also maybe you could add a try-catch block around it if for some reason the app fails to connect to the Database.